src/test/groovy/com/twcable/grabbit/jcr/JCRNodeDecoratorSpec.groovy

```diff
@@ -27,11 +28,22 @@ import javax.jcr.nodetype.ItemDefinition
 import javax.jcr.nodetype.NodeDefinition
 import javax.jcr.nodetype.NodeType
 
+import static org.apache.jackrabbit.JcrConstants.JCR_CREATED
 import static org.apache.jackrabbit.JcrConstants.JCR_LASTMODIFIED
 import static org.apache.jackrabbit.JcrConstants.JCR_PRIMARYTYPE
 
 @SuppressWarnings("GroovyAssignabilityCheck")
 class JCRNodeDecoratorSpec extends Specification {
+    @Shared
```

Comment:
Since these are only used in the bottom spec, these can probably just go there. 

With some exceptions, I'm generally not a fan of sharing resources between specs. It can open up a door to test isolation issues.